src/modules/tests/test-data/cat-d/_tests_/test-data.cat-d.selector.spec.ts

```diff
@@ -0,0 +1,390 @@
+import { CatDUniqueTypes } from '@dvsa/mes-test-schema/categories/D';
+import {
+  hasSeriousFault,
+  hasDangerousFault,
+  getETAFaultText,
+  getEcoFaultText,
+  getShowMeQuestionOptions,
+} from '../../common/test-data.selector';
+import {
+  getDrivingFaultCount,
+  getManoeuvres,
+  hasManoeuvreBeenCompletedCatD,
+  areTellMeQuestionsSelected,
+  areTellMeQuestionsCorrect,
+  hasVehicleChecksBeenCompletedCatD,
+} from '../test-data.cat-d.selector';
+import { Competencies } from '../../test-data.constants';
+import { CompetencyOutcome } from '../../../../../shared/models/competency-outcome';
+import { OutcomeBehaviourMapProvider } from '../../../../../providers/outcome-behaviour-map/outcome-behaviour-map';
+import { behaviourMap } from '../../../../../pages/office/office-behaviour-map';
```

Comment:
there is a cat d specific behaviour map. It's pretty much a copy of Cat B+E and Cat C but is the correct one to use for Cat D pages. (look in office page cat d for it's location)

Reply:
Looks like the test data selector needs a minor refactor. I have removed some tests which are not applicable to categories other than B and we should pick up the refactor outside of this PR.